knora/dsplib/models/propertyclass.py

```diff
@@ -417,7 +417,7 @@ def delete(self, last_modification_date: LastModificationDate) -> LastModificati
 
     def createDefinitionFileObj(self, context: Context, shortname: str):
         """
-        Create an object that jsonfied can be used as input to "create_onto"
+        Create an object that can be used as input to create an ontology on a DSP server
```

Comment:
"create an ontology on a DSP server" is very general. I always find it useful if I get precise directives like a method name. Or is that considered bad style?

Reply:
I don't have a strong opinion on that. I think clear instructions which method works with which can be helpful if input/output is very specific. In this case, where the output is according to our API specifications, it may make sense to keep it more generic.  
And I would normally keep the reference rather at the receiving method than on the producing method. But that may be another matter of taste.  
In any case, if we go for referencing methods directly, I suggest using back ticks to it's formatted as code, and maybe even write "`create_onto()`" to indicate that is's a method.

Reply:
Thanks for your thoughts, I now wrote `create_onto()`, so it's clear that it is a method.